#!/usr/bin/env python

PACKAGE='usv_estiplan'
import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator();

gen.add("r_xy", double_t, 1, "r_xy", 0.1, 0.0, 20.0);
gen.add("r_z", double_t, 0, "r_z", 0.1, 0.0, 20.0);
gen.add("r_heading", double_t, 0, "r_heading", 0.05, 0.0, 20.0);
gen.add("q_xy", double_t, 0, "q_xy", 10.2, 0.0, 20.0);
gen.add("q_z", double_t, 0, "q_z", 0.01, 0.0, 20.0);
gen.add("q_heading", double_t, 0, "q_heading", 0.1, 0.0, 20.0);
gen.add("q_v_xy", double_t, 0, "q_v_xy", 10.5, 0.0, 20.0);
gen.add("q_v_z", double_t, 0, "q_v_z", 0.01, 0.0, 20.0);
gen.add("q_heading_rate", double_t, 0, "q_heading_rate", 10.2, 0.0, 20.0);
gen.add("q_heading_rate_with_velocity", double_t, 0, "q_heading_rate_with_velocity", -0.3, -20.0, 20.0);
gen.add("water_drag", double_t, 0, "water_drag", 1.0, 0.0, 20.0);



exit(gen.generate(PACKAGE, "usv_estiplan", "dynparam"))

